docs/identities/sign-in/identifier-first-authentication.mdx

```diff
@@ -7,6 +7,12 @@ sidebar_label: Identifier first authentication
 Identifier first authentication first requests the user's identifier such as an email or username before prompting for a password
 or other authentication methods.
 
+:::note
+
+Identifier first authentication is required when using B2B Organization login.
+
+:::
```

Comment:
This note should be on the orgs docs, I think.

Reply:
I think in both, no?

Reply:
Well, ideally this would be in the product itself and does not require the user to know what knobs to turn!

Reply:
added the note